packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts

```diff
@@ -188,6 +188,13 @@ export class ForcedTradeOfferRepository extends BaseRepository {
     return rowIds.map(String).map(AssetId)
   }
 
+  async getByPositionId(position_id_a: bigint) {
+    const rows = await this.knex('forced_trade_offers')
+      .where({ position_id_a })
+      .whereNull('transaction_hash')
```

Comment:
```suggestion
      .whereNull('transaction_hash')
      .whereNull('cancelled_at')
```

We don't want to show cancelled offers.